### ISSUE TITLE
im vary new to python so can u help with this message

### ISSUE THREAD

**Issue by Deyelk** [YasserGersy/FacebookCracker#1]
raceback (most recent call last):
  File "FbCracker.py", line 347, in <module>
    info()
  File "FbCracker.py", line 214, in info
    ids="     ["+stx.Green+listtostr(idlist)+stx.yel+", "
TypeError: cannot concatenate 'str' and 'NoneType' objects

**Comment by YasserGersy** [comment]
this is a bug , thanks  for submitting it .
But unfortunately  this script is not done well yet , facebook prevents against brute force i'm still working around it , to find a way to bypass this protection , by the way i fixed the error , finally this script will fail after about 30 try so i do not think it will help you to get someone's password

**Comment by Deyelk** [comment]
cool I'm gonna read up and see if I can contribute to this script because it seems very interesting btw I'm new to Python

**Comment by Deyelk** [comment]
also do u have any ideas on bypassing the bruteforce protection

**Comment by YasserGersy** [comment]
Yes , i have but implementation needs much effort and may cost money :D

**Comment by Deyelk** [comment]
well much luck to yah

**Comment by Deyelk** [comment]
can u post here when its working

**Comment by YasserGersy** [comment]
OK, i think if i did it , i will notify you.
